apps/chat/src/components/Common/FolderContextMenu.tsx

```diff
@@ -140,27 +136,16 @@ export const FolderContextMenu = ({
         onClick: onPublishUpdate,
         disabled: disableAll,
       },
-      {
```

Comment:
Why did you remove it?

Reply:
I deleted it during development. I'll get it back when I make a folders unpublishing ticket

Reply:
Then just comment it with TODO-comment to not lose